server/liveblog/themes/themes.py

```diff
@@ -70,9 +75,27 @@ def get_local_themes_packages(self):
             for file in glob.glob(embed_folder + '/**/theme.json'):
                 yield json.loads(open(file).read())
 
+    def get_themes(self):
+        res = get_resource_service('themes').get(req=None, lookup={})
+        return res
+
     def update_registered_theme_with_local_files(self):
```

Comment:
I don't think this the best place to extract zip files.
You should extract it when the user uploads it.
Also, it's not necessary to extract it  in the local file system. In the media storage would be better because we use S3.